docs/user/container.md

```diff
@@ -366,8 +366,8 @@ This includes replicated or erasure-coded objects. Attempts to open an
 object with a class that does not support data redundancy (e.g., SX)
 will fail.
 
-For rf2, only objects with at least 3-way replication or erasure code with two
-parities or more can be stored in the container.
+For rd\_fac2, only objects with at least 3-way replication or erasure code with
```

Comment:
probably better here to just spell it out: redundancy factor 2

Reply:
OK I will going to fix this.

- [ ] Replace acronyme